creating_packages/define_abi_compatibility.rst

```diff
@@ -39,7 +39,7 @@ If the package is installed again with different settings, for example, for 32bi
 
 .. code-block:: bash
 
-    $ conan install MyLib/1.0@user/channel -s arch=x86 -s ...
+    $ conan install . MyLib/1.0@user/channel -s arch=x86 -s ...
```

Comment:
same here?

Reply:
True, you either install the path "." or the package "MyLib/1.0@user/channel"